Inc/Audio.h

```diff
@@ -491,23 +487,27 @@ namespace DirectX
     {
         float       EmitterAzimuths[XAUDIO2_MAX_AUDIO_CHANNELS];
 
-        AudioEmitter() throw()
-        {
-            memset(this, 0, sizeof(X3DAUDIO_EMITTER));
-            memset(EmitterAzimuths, 0, sizeof(EmitterAzimuths));
-
-            OrientFront.z = -1.f;
-
-            OrientTop.y =
-                ChannelRadius =
-                CurveDistanceScaler =
-                DopplerScaler = 1.f;
-
-            ChannelCount = 1;
-            pChannelAzimuths = EmitterAzimuths;
-
-            InnerRadiusAngle = X3DAUDIO_PI / 4.0f;
-        }
+        AudioEmitter() noexcept :
+            X3DAUDIO_EMITTER{
```

Comment:
This structure definition is hard to read.  I liked the original version of the code better - it was self documenting by seeing what field names are being assigned to.

Reply:
I can do it, but ``/analyze`` will go back to complaining...

Reply:
Isn't human readability > making analyzers happy? :-)